src/infection-model/rt.tsx

```diff
@@ -126,6 +127,21 @@ export const getRtDataForFacility = async (
   }
 };
 
+export async function getUpdatedFacilityRtData(
```

Comment:
nitpicking but I think a function that starts with "get" implies that it will return something; would just call this "updateFacilityRtData" or something like that

Reply:
Updated